.bldr.toml

```diff
@@ -843,6 +843,21 @@ plan_path = "rq"
 plan_path = "rsync"
 [ruby]
 plan_path = "ruby"
+[ruby24]
+plan_path = "ruby24"
+paths = [
+  "ruby/*"
+]
+[ruby23]
+plan_path = "ruby23"
+paths = [
+  "ruby/*"
+]
+[ruby22]
```

Comment:
Er ... well, I was silly and put these in descending version order because that's how I think of them. It looks odd here now that I see it.